yarn.lock

```diff
@@ -21131,7 +21136,7 @@ rsvp@^4.8.4:
   resolved "https://registry.yarnpkg.com/rsvp/-/rsvp-4.8.5.tgz#c8f155311d167f68f21e168df71ec5b083113734"
   integrity sha512-nfMOlASu9OnRJo1mbEk2cz0D56a1MBNrJ7orjRZQG10XDyuvwksKbuXNp6qa+kbn839HwjwhBzhFmdsaEAfauA==
 
-rtl-css-js@^1.1.3:
+rtl-css-js@^1.1.3, rtl-css-js@^1.14.0:
```

Comment:
We are getting a duplicated entry here, maybe it would be good to go and look for where else are we importing `rtl-css-js` and bump the dependency if we can.

Reply:
`rtl-css-js` is a dependency of `fela-plugin-rtl`, I don't think that we should bump anything as it's not a duplicate dependency. See examples below:

---

This entry matches both versions and will be resolved to a single package to be installed.

```
rtl-css-js@^1.1.3, rtl-css-js@^1.14.0:
  version "1.14.0"
```

However, this PR intentionally contains Stylis v4 (we use in Northstar v3) which results in two packages to be installed:

```
stylis@^3.5.4:
  version "3.5.4"

stylis@^4.0.6:
  version "4.0.6"
```